src/pages/settings/PasswordPage.js

```diff
@@ -1,20 +1,152 @@
-import React from 'react';
+import React, {Component} from 'react';
+import {View, TextInput} from 'react-native';
+import Onyx, {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import {isEmpty} from 'underscore';
+
 import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
 import Navigation from '../../libs/Navigation/Navigation';
 import ROUTES from '../../ROUTES';
 import ScreenWrapper from '../../components/ScreenWrapper';
+import Text from '../../components/Text';
+import styles from '../../styles/styles';
+import ONYXKEYS from '../../ONYXKEYS';
+import ButtonWithLoader from '../../components/ButtonWithLoader';
+import {changePassword} from '../../libs/actions/User';
+
+const propTypes = {
+    /* Onyx Props */
+    // Holds information about the users account that is logging in
+    account: PropTypes.shape({
+        // An error message to display to the user
+        error: PropTypes.string,
+
+        // Whether or not a sign on form is loading (being submitted)
+        loading: PropTypes.bool,
+    }),
+};
+
+const defaultProps = {
+    account: {},
+};
+
+// at least 8 characters, 1 capital letter, 1 lowercase number, 1 number
+const passwordComplexityRegexString = '^(?=.*[A-Z])(?=.*[0-9])(?=.*[a-z]).{8,}$';
```

Comment:
Let's move this CONST.js maybe? I imagine we'll need to reuse it soon-ish anyway.